nservicebus/messaging/delayed-delivery.md

```diff
@@ -5,10 +5,21 @@ tags:
 - Defer
 ---
 
-When a message is sent it does not need to be dispatched immediately. You can specify a later time for the message to be delivered. Before version 6 this was achieved by using `bus.Defer(...)` instead of `bus.Send(...)`. In version 6, this is accomplished by calling methods on an instance of `SendOptions` that is passed into `Send(...)` calls.  
+When a message is sent it does not need to be dispatched immediately. You can specify a later time for the message to be delivered. Before version 6 this could be achieved by using `bus.Defer(...)` instead of `bus.Send(...)`. In version 6, this is accomplished by calling methods on an instance of `SendOptions` that is passed into `Send(...)` calls.  
```

Comment:
remove the personal voice. "you". perhaps rather than the inline code we should have a snippet?